### PR TITLE
multilocation handling

### PR DESCRIPTION
**Pull request by KwanyoungLee**

### CODE REVIEW

File: src/components/Case.js [hunk 1]
```diff
@@ -108,12 +113,14 @@ const Case = () => {
...
+        setDbOutput(res.data.location_db);
+        setLocationOutput(res.data.location_geo);
       })
       .catch((error) => {
-        if (error.response.status === 400) {
-          alert("No search result");
-        }
+        //if (error.response.status === 400) {
+          //alert("No search result");
+        //}
```
Comment (BobbyLeeSH) [inline src/components/Case.js]: please remove the commented out lines!

File: src/components/Case.js [hunk 2]
```diff
@@ -108,12 +113,14 @@ const Case = () => {
         params: { q: locationInput },
       })
       .then((res) => {
-        setLocationOutput(res.data[0]);
+        console.log(res.data);
```
Comment (kau96kim) [inline src/components/Case.js:116]: console log on live code..? O M G

**Review by BobbyLeeSH — APPROVED** [review]
LGTM!!!!!!!!!

File: src/components/Case.js [hunk 3]
```diff
@@ -243,11 +301,15 @@ const Case = () => {
                 />
               </Td>
               <Td>
-                <Input
+                <select
```
Comment (kau96kim) [inline src/components/Case.js]: **_so good~!_**

File: src/components/Case.js [hunk 4]
```diff
@@ -212,16 +246,40 @@ const Case = () => {
...
-              <Td>{locationOutput.addressEN}</Td>
-              <Td>{locationOutput.x}</Td>
-              <Td>{locationOutput.y}</Td>
-            </tr>
+                <Th>Location (Recently added)</Th>
+                <Th>Address</Th>
+                <Th>X Coord</Th>
+                <Th>Y Coord</Th>
+                <Th></Th>
+                {dbOutput.map((output, index) => (
```
Comment (kau96kim) [inline src/components/Case.js:271]: Who's idea is this code? This is **_### genius_**

**Pull request merged**